.github/workflows/push-asset-readme-update.yml

```diff
@@ -0,0 +1,17 @@
+name: Plugin assets/readme update
+on:
+  push:
+    branches:
+    - trunk
+jobs:
+  trunk:
+    name: Deploy assets/readme to WP.org
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: WordPress.org plugin asset/readme update
+      uses: 10up/action-wordpress-plugin-asset-update@stable
+      env:
+        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+        SLUG: debug-bar-elasticpress
```

Comment:
note that you don't need the SLUG variable if that already matches the repo name/capitalization